pyrobuf/protobuf/templates/proto_pxd.tmpl

```diff
@@ -39,7 +39,7 @@ cdef class {{ message.full_name }}:
     cdef public bint _is_present_in_parent
     cdef bytes _cached_serialization
 
-    cdef PyObject *_listener
+    cdef object _listener
```

Comment:
I wonder if there's a way to type this as a function or something, but I think the simplicity is maybe worth any resulting perf hit.

Reply:
Not with an arbitrary bound reference as far as I can tell; you'd certainly need two pointers (function, object). I think most of the speed hit comes from this change.